nautobot-app/{{ cookiecutter.project_slug }}/docs/admin/install.md

```diff
@@ -21,7 +21,7 @@ Here you will find detailed instructions on how to **install** and **configure**
 ## Install Guide
 
 !!! note
-    Apps can be installed manually or using Python's `pip`. See the [nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this app is [`{{ cookiecutter.app_slug }}`](https://pypi.org/project/{{ cookiecutter.app_slug }}/).
+    Apps can be installed manually or using Python's `pip`. See the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this app is [`{{ cookiecutter.app_slug }}`](https://pypi.org/project/{{ cookiecutter.app_slug }}/).
```

Comment:
This should probably be a docs.nautobot.com link instead, shouldn't it? Probably https://docs.nautobot.com/projects/core/en/stable/user-guide/administration/installation/app-install/

Reply:
@whitej6 WDYT?

Reply:
The current link is more for app developers than deployers/administrators, which is why I suggested the alternate link that's specifically for deployments.